src/FormBuilder/resources/js/form-builder/src/promos/additionalFields/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {__} from '@wordpress/i18n';
+import LockedFieldBlocks, {LockIcon} from './LockedFieldBlocks';
+import './styles.scss';
+
+/**
+ * @unreleased
+ */
+export default function AdditionalFieldsPanel() {
+    return (
+        <>
+            <div className={'block-editor-inserter__panel-header'}>
+                <h2
+                    className={
+                        'block-editor-inserter__panel-title block-editor-inserter__panel-additional-fields-header'
+                    }
+                >
+                    {__('Additional Fields', 'give')}
+                    <LockIcon />
+                </h2>
+            </div>
+
+            <div className="block-editor-inserter__panel-content">
+                <a
```

Comment:
We should add `rel="noopener noreferrer"` for external links.

Reference: https://lw.slack.com/archives/C02QJ8F07B3/p1668803646241939?thread_ts=1668790561.688639&cid=C02QJ8F07B3

Reply:
Resolved: 2a9857e61a638d7873b224f9150d298d3dd22b7e